Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM node:boron
 
 # Update and install packages
 RUN apt-get update && \
-    apt-get install -y python python-dev python-pip python-virtualenv zip rsync && rm -rf /var/lib/apt/lists/*
+    apt-get install -y python python-dev python-pip python-virtualenv zip rsync default-jdk && \
```

Comment:
Tried many things, but couldn't find a solution!
* default-jdk=2:1.8-56ubuntu2 (version not found)
* Tried 
![image](https://user-images.githubusercontent.com/8245662/29329655-334ec21e-8214-11e7-8744-f033cfefbb76.png)
Couldn't find curl
Couldn't find add-apt-repository
Couldn't find software-properties-common